pkg/manifests/manifests.go

```diff
@@ -836,8 +837,22 @@ func (f *Factory) updateNodeExporterArgs(args []string) []string {
 		args = setArg(args, "--no-collector.tcpstat", "")
 	}
 
+	var excludedDevices string
+	if f.config.ClusterMonitoringConfiguration.NodeExporterConfig.Collectors.NetDev.Enabled ||
+		f.config.ClusterMonitoringConfiguration.NodeExporterConfig.Collectors.NetClass.Enabled {
+		devs := *f.config.ClusterMonitoringConfiguration.NodeExporterConfig.IgnoredNetworkDevices
+		if len(devs) > 0 {
```

Comment:
Do we need to a special case for zero-length list?

Reply:
well, an empty list would generate this: `--collector.netclass.ignored-devices=^()$` ... which technically could work... that would only match empty strings, which I guess is not supposed to happen for a network device label? I'm not 100% sure

Anyway it seems to me that removing this command line argument would be cleaner and more clear, than passing a regex that matches nothing?

Reply:
I don't feel strongly on this one but at least a comment would help for the future :)

Reply:
comment done